app/presenters/two_factor_auth_code/phone_delivery_presenter.rb

```diff
@@ -56,10 +56,10 @@ def troubleshooting_options
           text: troubleshoot_change_phone_or_option_text
         },
         {
-          url: MarketingSite.help_center_article_url(
+          url: "#{MarketingSite.help_center_article_url(
             category: 'get-started',
             article: 'authentication-options',
-          ),
+          )}#text-message--phone-call",
```

Comment:
This is probably worth moving into the `MarketingSite` class in some form as the anchor might be useful to use elsewhere.